configs/restorers/rdn/rdn_x2c64b16_g1_1000k_div2k.py

```diff
@@ -0,0 +1,124 @@
+exp_name = 'rdn_x2c64b16_g1_1000k_div2k'
+
+scale = 2
+# model settings
+model = dict(
+    type='BasicRestorer',
+    generator=dict(
+        type='RDN',
+        in_channels=3,
+        out_channels=3,
+        mid_channels=64,
+        num_blocks=16,
+        upscale_factor=scale),
+    pixel_loss=dict(type='L1Loss', loss_weight=1.0, reduction='mean'))
+# model training and testing settings
+train_cfg = None
+test_cfg = dict(metrics=['PSNR', 'SSIM'], crop_border=scale)
+
+# dataset settings
+train_dataset_type = 'SRAnnotationDataset'
+val_dataset_type = 'SRFolderDataset'
+train_pipeline = [
+    dict(
+        type='LoadImageFromFile',
+        io_backend='disk',
+        key='lq',
+        flag='unchanged'),
```

Comment:
Effect of 'unchanged': if it is grayscale image, you get 1 channel; if it has transparency, you get 4 channels; if it contains orientation metadata, it ignores.

Is this intended?

Reply:
No, this is not intended.